test/test-suite-whitelist.jsonc

```diff
@@ -82,6 +82,7 @@
       "test_flatten_default_axis",
       "test_gather_0",
       "test_gather_1",
+      "test_scatter_0",
```

Comment:
this test does not exist. please see https://github.com/onnx/onnx/tree/master/onnx/backend/test/data/node